ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -488,18 +489,26 @@ aaa_server_groups:
 ```yaml
 aaa_authentication:
   login:
-    default: < group | local | none >
-    serial_console: < group | local | none >
+    default: < group name >  | local | none >
+    serial_console: < group name >  | local | none >
   dot1x:
-    default: < group | local | none >
+    default: < group name > | local | none >
+  policies:
+    on_failure_log: < true | false >
+    on_success_log: < true | false >
+    local:
+        allow_nopassword: < false | true (default) >
 ```
 
 ### AAA Authorization
 
 ```yaml
 aaa_authorization:
-  exec_default: < group | local | none >
+  exec_default: < group name > | local | none >
```

Comment:
So we have 3 different models in authentication, authorization and accounting.
```
login:
  default: < group name | local | none >
```
vs
```
exec_default: < group name | local | none >
```
vs
```
exec:
  default:
    group: <group name>
```

I know  you are only adding to existing models, but it is becoming very confusing!

Reply:
Data model change will be taken into account.
The `exec_default: < group name | local | none >` will be changed to

```
exec:
    default: < group name | local | none >
```

@ClausHolbechArista

Reply:
awesome! Thanks.

Reply:
Resolved.